service/src/main/java/uk/gov/hmcts/reform/fpl/service/DocumentUploadHelper.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.enums.UserRole;
+import uk.gov.hmcts.reform.fpl.request.RequestData;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+@Service
+@RequiredArgsConstructor(onConstructor_ = {@Autowired})
+public class DocumentUploadHelper {
```

Comment:
Not sure I like a helper being a service, to me a service is something which should contain business logic and helpers should be a reusable class that don't need any fancy setup. Maybe a better approach would be to have the user details passed to the method?

Reply:
I had this as a service before and forgot to change annotation. I will change that to Component. If i am passing user details I have to add IdamClient and getUserDetails in all the places i am using it. Let me know what you think!

Reply:
Component is definitely better, I would move it out of the service package as well

Reply:
done